test/iterator/iterator.cpp

```diff
@@ -33,13 +33,23 @@ struct reference_wrapper {
 		return *ptr_;
 	}
 
-	reference_wrapper& operator=(const T& t)
+	reference_wrapper & operator=(const T& t)
 		noexcept(std::is_nothrow_copy_assignable<T>::value) {
 		get() = t;
 		return *this;
 	}
+	reference_wrapper & operator=(T&& t)
+		noexcept(std::is_nothrow_move_assignable<T>::value) {
+		get() = __stl2::move(t);
+		return *this;
+	}
 
-	reference_wrapper& operator=(T&& t)
+	reference_wrapper const& operator=(const T& t) const
+		noexcept(std::is_nothrow_copy_assignable<T>::value) {
+		get() = t;
+		return *this;
+	}
+	reference_wrapper const& operator=(T&& t) const
```

Comment:
Same comment: we don't need to satisfy `Assignable`; I think the non-`const` overloads are extraneous.